deploy/charts/emqx/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 5.3.2
+version: 5.3.2-alpha.2
```

Comment:
Just a note: I think that if `EMQX_RELEASE_CE` didn't change, this one doesn't need bumping.  :thinking: